src/frontend/src/handler/create_mv.rs

```diff
@@ -86,15 +86,15 @@ pub fn gen_create_mv_plan(
 
     let definition = context.normalized_sql().to_owned();
 
-    let bound = {
+    let (dependent_views, bound) = {
         let mut binder = Binder::new_for_stream(session);
         let bound = binder.bind_query(query)?;
-        // FIXME: We should record the views into mv's dependent relations to to refine the drop
-        // check and recursive rename of the views.
-        let _views = binder.shared_views().keys().cloned().collect_vec();
-        bound
+        (binder.shared_views(), bound)
     };
 
+    let check_items = resolve_query_privileges(&bound);
+    session.check_privileges(&check_items)?;
```

Comment:
What's this change about? 👀

Reply:
The logic for checking privileges was missing here previously.